test/Microsoft.VisualStudio.Sdk.TestFramework.Tests/SomePackage.cs

```diff
@@ -23,8 +23,8 @@ public interface SVsNonPromotedService
 
     protected override Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
     {
-        this.AddService(typeof(SVsPromotedService), (sc, ct, serviceType) => Task.FromResult<object?>(new object()), true);
-        this.AddService(typeof(SVsNonPromotedService), (sc, ct, serviceType) => Task.FromResult<object?>(new object()), false);
+        this.AddService(typeof(SVsPromotedService), (sc, ct, serviceType) => Task.FromResult(new object()), true);
+        this.AddService(typeof(SVsNonPromotedService), (sc, ct, serviceType) => Task.FromResult(new object()), false);
```

Comment:
@davkean this change is interesting. Does this mean that `AsyncPackage.AddService` service factories are not allowed to return null? Or is this a case of a missing nullable annotation when you turned on `#nullable enable` [in this change](https://dev.azure.com/devdiv/DevDiv/_git/VS/commit/2f86872854120e1aa7801f25d5b0c4c484e69a3e?refName=refs%2Fheads%2Fmain&_a=compare&path=%2Fsrc%2Fenv%2Fshell%2FPackageFramework%2FFramework%2FIAsyncServiceContainer.cs)?